### ISSUE TITLE
Button Press only works one time on HmIP-RCV-50 HmIP-RCV-1 

### ISSUE THREAD

**Issue by Aggro600** [SukramJ/hahomematic#938]
This is for issues only
If you have a feature request, add it as a new discussion, or add your point of view to an existing discussion.

# DON'T DELETE THIS.

Issue without this template will be closed.

## Please answer all questions as good as possible.

Language: as you like -> english/DEUTSCH

Make sure you have read the documentation at https://github.com/danielperna84/custom_homematic#custom_homematic.

Make sure you have read the release notes at https://github.com/danielperna84/custom_homematic/releases.

Make sure you are running the latest version of the custom_component (and Home Assistant) before reporting an issue.

ENSURE that your Backend (CCU/Homegear/...) is working as expected and connected devices are controllable.

**custom_component version (if applicable):**
1.x.x

**known last working custom_component version (if applicable):**
1.x.x

**Home Assistant:**

- Version: 2023.2.2
- Installation method (HAOS, supervised, core, venv, Docker): HAOS

**CCU:**

- Model (CCU2/3, Raspi3/4, NUC, ProxmoxVE, HA-Addon ...): Raspi4
- OS (Original, RaspberryMatic, debmatic ...): RaspberryMatic addon for homeassistant
- Version:

**Problem-relevant configuration:**

Do you use tls? no
Do you use callback? no
Do you use username and password? yes
Which interfaces do you use (hmip/bidcos/wired)? hmip

**Describe the bug**

**To Reproduce**

1. Go to 'HmIP-RCV-50 HmIP-RCV-1 in Homeassistant'
2. Click on 'Any channel ( for example button.hmip_rcv_50_hmip_rcv_1_press_long_ch1'
3. Go to 'CCU>Device>HmIP-RCV-50 HmIP-RCV-1'
4. See error Only the first button press is fired.

**Screenshots (if applicable)**

- CCU firewall in CCU
- device config in CCU
- integrations view in HA
- device view in HA
- entity view in HA

**Relevant HA log entries (press full log) VERY IMPORTANT**

```
No Eorros
XXX wechselte zu 7. Februar 2023 um 17:19 ausgelöst durch Dienst button.press

```

**Diagnostic Information (available on every device)**

```


```

**Expected behavior**

**Additional context**

**Comment by SukramJ** [comment]
You can not enter the versions used?

**Comment by SukramJ** [comment]
Thanks for reporting. Fixed with 1.30.1.

It would be nice if you could also add the corresponding versions next time.
